public/pages/Snapshots/components/RenameInput/RenameInput.tsx

```diff
@@ -9,11 +9,15 @@ import { RESTORE_SNAPSHOT_DOCUMENTATION_URL } from "../../../../utils/constants"
 interface RenameInputProps {
   getRenamePattern: (prefix: string) => void;
   getRenameReplacement: (prefix: string) => void;
+  showPatternError: boolean;
+  showRenameError: boolean
 }
 
-const RenameInput = ({ getRenamePattern, getRenameReplacement }: RenameInputProps) => {
+const RenameInput = ({ getRenamePattern, getRenameReplacement, showPatternError, showRenameError }: RenameInputProps) => {
   const [renamePattern, setRenamePattern] = useState("(.+)");
   const [renameReplacement, setRenameReplacement] = useState("restored_$1");
+  const badRenamePattern = "Please enter a valid Rename pattern.";
+  const badRenameReplacement = "Please enter a valid Rename replacement."
```

Comment:
These should either be memoized or moved to a constants file since they get recreated on every render and could lead to unnecessary re-renders downstream

Reply:
similarly for other constant strings

Reply:
I continue to learn from you.  Changed for next PR